packages/venia-concept/src/RootComponents/Category/categoryContent.js

```diff
@@ -21,13 +45,26 @@ class CategoryContent extends Component {
                         }}
                     />
                     <div className={classes.categoryTitle}>{categoryTitle}</div>
+                    <div className={classes.headerButtons}>
+                        <button
+                            onClick={openDrawer}
```

Comment:
For the same reason, could we just `mapDispatchToProps` here instead of having to thread down the handler?

Reply:
Fixed in latest commit.